Open machine listing when starting webpack-dev-server.

squidsoup opened the pull request:
## Done

- Open machine listing when starting webpack-dev-server.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Start dev server, should open to machine listing.

## Backports

In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in master.

huwshimi commented:
I think this also needs a redirect from /MAAS to /MAAS/r/ (I think that was the intention of the original issue).

huwshimi commented:
I've added a fix for #1265 in https://github.com/canonical-web-and-design/maas-ui/pull/1454

squidsoup commented:
> I've added a fix for #1265 in #1454

Ah thanks, I've reverted the redirect fix in this branch, so this only now opens the machine listing on starting the devserver.
